src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/ConfirmProviderViewModelMapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.Commitments.Shared.Interfaces;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Mappers
+{
+    public class ConfirmProviderViewModelMapper : IMapper<ConfirmProviderRequest, ConfirmProviderViewModel>
+    {
+        public ConfirmProviderViewModel Map(ConfirmProviderRequest source)
+        {
+            return new ConfirmProviderViewModel
+            {
+                AccountHashedId = source.AccountHashedId,
+                EmployerAccountLegalEntityPublicHashedId = source.EmployerAccountLegalEntityPublicHashedId,
```

Comment:
this should be AccountLegalEntityHashedId

Reply:
This is fixed in #29